moneytracker/models.py

```diff
@@ -34,6 +43,7 @@ class PortfolioAsset(models.Model):
     acquisition_date = models.DateField(auto_now=True)
 
     class Meta:
+        db_table = 'portfolios_asset'
```

Comment:
dude:
its 'portfolio_asset' or 'portfolio_assets' and why?

Reply:
I'm not even sure how it should be in Spanish, but 'portfolios_assets' sounds better